fleetspeak/src/e2etesting/setup/setup_components.go

```diff
@@ -220,7 +220,7 @@ func buildBaseConfiguration(configDir string, mysqlCredentials MysqlCredentials,
 	return nil
 }
 
-func modifyFleetspeakServerConfig(configDir string, fsServerHost string, fsFrontendPort, fsAdminPort int, HTTPSListenAddress, newServerConfigPath, newServerServicesConfigPath string) error {
+func modifyFleetspeakServerConfig(configDir string, fsServerHost string, fsFrontendPort, fsAdminPort, fsHealthCheckPort int, HTTPSListenAddress, newServerConfigPath, newServerServicesConfigPath string) error {
```

Comment:
This is starting to have too many arguments, maybe it should get some struct or structs instead (similar yo MysqlCredentials).